README.md

```diff
@@ -1,7 +1,7 @@
-# <img alt="ORT Server" src="assets/logo.svg" width="10%"> ORT Server 
+# <img alt="ORT Server" src="assets/logo.svg" width="10%"> Eclipse Apoapsis - ORT Server 
 
-The ORT Server is a standalone application to deploy the
-[OSS Review Toolkit](https://github.com/oss-review-toolkit/ort) as a service in the cloud.
+The [Eclipse Apoapsis](https://projects.eclipse.org/projects/technology.apoapsis) ORT Server is a standalone application
```

Comment:
Maybe: "The ORT Server of the Eclipse Apoapsis project..."?

Reply:
I have used the slightly different "The Eclipse Apoapsis project's ORT Server".